testing/otel-verify/src/index.ts

```
@@ -0,0 +1,124 @@
+ *
+ * Wires `@opentelemetry/sdk-node` with an OTLP/HTTP trace exporter pointed at
+ * one backend preset (see `backends.ts`), boots an in-process aimock for
+ * deterministic LLM responses, and runs the scenarios in `scenarios.ts`. Each
+ * scenario triggers `otelMiddleware` end-to-end so the resulting spans/events/
+ * histograms land on the configured backend.
+ *
+ * Filter scenarios with `SCENARIO=<id>` (comma-separated) — e.g.
+ * `SCENARIO=basic-text,error` to skip the tool round-trip while smoke-testing.
+ */
+
+import { trace, metrics } from '@opentelemetry/api'
```

ESLint

[error] Member 'metrics' of the import declaration should be sorted alphabetically.

(sort-imports)